editor/src/clj/editor/workspace.clj

```diff
@@ -813,7 +813,7 @@ ordinary paths."
 
 (defn resource-kind-extensions [workspace resource-kind]
   (let [extensions-by-resource-kind (g/node-value workspace :resource-kind-extensions)]
-    (or (extensions-by-resource-kind resource-kind)
+    (or (extensions-by-resource-kind (keyword resource-kind))
```

Comment:
Since the script properties uses string "atlas" as opposed to the keyword `:atlas`.

Not sure if there is a better way of doing it?